src/sentry/static/sentry/app/views/performance/trends/chart.tsx

```diff
@@ -109,6 +109,10 @@ function getLegend() {
         name: 'Releases',
         icon: 'line',
       },
+      {
+        name: trendFunction,
```

Comment:
Not sure using the trend function as the label communicates the intent very well 🤔

Reply:
Hmm any suggestions otherwise? the tooltip currently shows `p50()` for the raw graph.... `unsmoothed` or `raw graph` both sound a bit awkward to me

<img width="206" alt="image" src="https://user-images.githubusercontent.com/4205004/96903087-61334580-1463-11eb-97d4-2236ff33c789.png">

Reply:
Hm... can the tooltip here use different colors for `p50()` and `smoothed p50()`? At least that way, it's possible to see which is which.

Reply:
<img width="868" alt="image" src="https://user-images.githubusercontent.com/4205004/96939359-d8380080-149a-11eb-8910-20f3ded73397.png">
<img width="871" alt="image" src="https://user-images.githubusercontent.com/4205004/96939367-dcfcb480-149a-11eb-9402-0c790c93edb1.png">

Reply:
Nice 👍